src/EmbeddedVsix/EmbeddedVsix.csproj

```diff
@@ -0,0 +1,115 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I've left this as an old-style project for now. The first 47 lines are boilerplate - the custom stuff is below that.